src/components/NcPopover/NcPopoverTriggerProvider.vue

```diff
@@ -34,8 +34,8 @@ export default defineComponent({
 	mounted() {
 		if (window.OC?.debug) {
 			const rootElement = this.$el
-			const innerElement = this.$el.querySelector('[aria-expanded][aria-haspopup]')
-			if (!rootElement.getAttribute('aria-expanded') && !innerElement) {
+			const innerElement = this.$el?.querySelector?.('[aria-expanded][aria-haspopup]')
+			if (!rootElement?.getAttribute?.('aria-expanded') && !innerElement) {
```

Comment:
Not important to the current PR, but have seen this in many places. Optional chaining `?.` works for a whole chain.

```js
const a1 = null
const a2 = { b: { c: { d() } } }
// This is enough to check for a1 == null
a1?.b.c.d()
a2?.b.c.d()
// This is unneeded and adds redundant huge code after transpiling
a1?.b?.c?.d?.()
```

Here it is guaranteed that if html element exists, it has `querySelector` and `getAttributes` methods.